### DISCUSSION TITLE
Rate limit on MJML API?

### DISCUSSION THREAD

**Discussion by Plaschke-Culturall** [mjmlio/mjml#2888] [General]
Hello!

We have been using the MJML API for some time now to easily format a lot of marketing and other e-mails. I wanted to ask if there is a rate limit on using the API and what it is? Would it be possible to e.g. send a request every 1 or 5 or 10 minutes to monitor the availability of the API or could that lead to our account being blocked?

Thank you for your Assistance!

**Comment by iRyusa** [comment]
There's no SLA on this API so you shouldn't rely on this for a production service imo. The MJML version is really outdated too, you'll miss a lot of fixes.
